data/elevation-and-terrain/2020-lidar-northern-central-utah/index.md

```diff
@@ -0,0 +1,44 @@
+---
+status: publish
+layout: page
+tags:
+  - dem
+  - lidar
+  - elevation
+author:
+  display_name: Rick Kelson
+  email: rkelson@utah.gov
+published: true
+date: 2020-07-05 12:30:30
```

Comment:
**question**: do we want to update this to match nowish or does it matter?

Reply:
yes, let me do that